### PR TITLE
Add CoC and License, point to ignite community resources for now

### PR DESCRIPTION
**Pull request by dholbach**

### CODE REVIEW

**Comment by luxas** [comment]
Thanks, could you also add @stealthybox and @chanwit as maintainers here?

**Comment by stealthybox** [comment]
^ ack

**Comment by dholbach** [comment]
You want me to list them in `CODE_OF_CONDUCT.md`? Or link to the MAINTAINERS file maybe?

File: README.md [hunk 1]
```diff
@@ -2,3 +2,13 @@
 
 A set of packages to help build Git-backed applications.
 Weave GitOps Toolkit builds on top of the [Kubernetes API Machinery](https://github.com/kubernetes/apimachinery).
+
+## Getting Help
+
+If you have any questions about, feedback for or problems with `ignite`:
```
Comment (palemtnrider) [inline README.md]: Are we planning to provide support for this tool via the ignite slack channel?  I think that's OK but we should consider changing this message to say something about that.  i.e. "for help with gitops-toolkit, post a message on the `ignite` channel in the Weave Users Slack..."

**Comment by dholbach** [comment]
I can't merge, can you do that?

**Comment by palemtnrider** [comment]
I don't have that permission either.  Let me sort that out.

**Pull request merged**